### PR TITLE
Update storage/butler descriptions.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

File: usdf/storage.rst [hunk 1]
```diff
@@ -8,40 +8,49 @@ Storage Locations
...
+ - send email to ``usdf-help`` at ``slac.stanford.edu`` requesting a personal web directory
+ - a ``public_html`` directory will be created in your home directory after acknowledging a usage policy
+ - note that this is for static pages - no server-side scripting.
+ - no symlinks outside ``public_html`` will be followed.
+
+Science data under ``/sdf/group/rubin/``:
+ - Much of this is still disorganized and contains vestiges of the structure from NCSA that was moved to USDF.  Bear with us until we get it all straightened out.
+ - ``shared/`` contains various datasets for common usage.
+   - ``shared/data-repos.yaml`` contains the list of aliases for Butler repos.
+   - shared/data contains daily-updated test and validation datasets.
```
Comment (hsinfang) [inline usdf/storage.rst:25]: Should this be ``shared/data/test_data`` ?

File: usdf/storage.rst [hunk 2]
```diff
@@ -8,40 +8,49 @@ Storage Locations
...
+
+Available Butler repos:
+   - ``main`` - All images taken of the real sky by HSC, DECam, LATISS, LSSTComCam, and LSSTCam.  (Post-embargo images for the Rubin cameras.)
+   - ``embargo`` - Embargoed data from Summit cameras (located in the Embargo Rack).
+   - ``prompt`` - Summit camera raw images and data products published as nightly post-embargo data products for LSST data rights holders.
+   - ``dc2`` - DESC DC2 simulated LSSTCam.
+   - ``ir2`` - LSSTCam and TS8 data taken at SLAC during testing.
+   - ``bts`` - Equivalent of ``embargo`` for the Base Test Stand (located in the Embargo Rack).
+   - ``tts`` - Equivalent of ``embargo`` for the Tucson Test Stand (not located in the Embargo Rack)
+   - ``ops-rehearsal-3-prep`` - Images and data products for Ops Rehearsals 3 and 4.  Data products generated as a result of the Ops Rehearsals are elsewhere, starting in ``/repo/embargo``..
```
Comment (hsinfang) [inline usdf/storage.rst:44]: Maybe... for Ops Rehearsals 3 and 4 "preparation"?

File: usdf/storage.rst [hunk 3]
```diff
@@ -8,40 +8,49 @@ Storage Locations
...
-Interim while NCSA data is being placed at SLAC
- - SDF Lustre is separate from s3df filesystems. Access the Lustre filesystem via ``/fs/ddn/sdf/`` to access your SDF home directories, sandbox and scratch files
- - datasets and repo/main are read-only, with the exception of repo/main_20220411, which is live and in use for HSC reprocessing
-
 Butler access
 =============
 
-The primary Butler repos are located at ``/sdf/group/rubin/repo/main_20210215`` and ``/sdf/group/rubin/repo/dc2_20210215``.
-They can also be accessed via aliases ``/repo/main`` and ``/repo/dc2``.
+Butler repo configurations are located under ``/sdf/group/rubin/repo/``, but we define aliases for them starting with ``/repo/`` as a convenience.
```
Comment (hsinfang) [inline usdf/storage.rst:34]: Nitpicky but the first part is not exactly true. Some of the (source-of-truth) repo configurations are at their S3 bucket locations, not under ``/sdf/group/rubin/repo/``

File: usdf/storage.rst [hunk 4]
```diff
@@ -8,40 +8,49 @@ Storage Locations
...
+Available Butler repos:
+   - ``main`` - All images taken of the real sky by HSC, DECam, LATISS, LSSTComCam, and LSSTCam.  (Post-embargo images for the Rubin cameras.)
+   - ``embargo`` - Embargoed data from Summit cameras (located in the Embargo Rack).
+   - ``prompt`` - Summit camera raw images and data products published as nightly post-embargo data products for LSST data rights holders.
+   - ``dc2`` - DESC DC2 simulated LSSTCam.
+   - ``ir2`` - LSSTCam and TS8 data taken at SLAC during testing.
+   - ``bts`` - Equivalent of ``embargo`` for the Base Test Stand (located in the Embargo Rack).
+   - ``tts`` - Equivalent of ``embargo`` for the Tucson Test Stand (not located in the Embargo Rack)
+   - ``ops-rehearsal-3-prep`` - Images and data products for Ops Rehearsals 3 and 4.  Data products generated as a result of the Ops Rehearsals are elsewhere, starting in ``/repo/embargo``..
+   - ``hsc_pdr2_multisite`` - Special campaign for testing multi-site Data Release Production (DRP).
```
Comment (hsinfang) [inline usdf/storage.rst:45]: This one doesn't have an alias defined

File: usdf/storage.rst [hunk 5]
```diff
@@ -8,40 +8,49 @@ Storage Locations
...
 Butler access
 =============
 
-The primary Butler repos are located at ``/sdf/group/rubin/repo/main_20210215`` and ``/sdf/group/rubin/repo/dc2_20210215``.
-They can also be accessed via aliases ``/repo/main`` and ``/repo/dc2``.
+Butler repo configurations are located under ``/sdf/group/rubin/repo/``, but we define aliases for them starting with ``/repo/`` as a convenience.
+
+Available Butler repos:
+   - ``main`` - All images taken of the real sky by HSC, DECam, LATISS, LSSTComCam, and LSSTCam.  (Post-embargo images for the Rubin cameras.)
+   - ``embargo`` - Embargoed data from Summit cameras (located in the Embargo Rack).
```
Comment (hsinfang) [inline usdf/storage.rst:38]: Do we want to use this chance to mention that expired data will be moved out?

**Review by hsinfang — APPROVED** [review]
Thanks for updating this!

File: usdf/storage.rst (hunk 1 above)
Comment (ktlim) [reply usdf/storage.rst:25]: Both `test_data` and `validation_data` are updated.  (The other stuff needs to be moved away.)

**Pull request merged**